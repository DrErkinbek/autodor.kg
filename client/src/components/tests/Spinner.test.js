import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Spinner from '../layout/Spinner';

it('Spinner Component renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Spinner />, div);
});

it('Spinner Component renders correctly and equal to snapshot', () => {
    const tree = renderer.create(<Spinner />).toJSON();
  
    expect(tree).toMatchSnapshot();
});