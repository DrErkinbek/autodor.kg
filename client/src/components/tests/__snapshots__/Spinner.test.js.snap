// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders correctly and equal to snapshot 1`] = `
<div
  className="container center"
>
  <div
    className="preloader-wrapper big active"
  >
    <div
      className="spinner-layer spinner-blue-only"
    >
      <div
        className="circle-clipper left"
      >
        <div
          className="circle"
        />
      </div>
      <div
        className="gap-patch"
      >
        <div
          className="circle"
        />
      </div>
      <div
        className="circle-clipper right"
      >
        <div
          className="circle"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Spinner Component renders correctly and equal to snapshot 1`] = `
<div
  className="container center"
>
  <div
    className="preloader-wrapper big active"
  >
    <div
      className="spinner-layer spinner-blue-only"
    >
      <div
        className="circle-clipper left"
      >
        <div
          className="circle"
        />
      </div>
      <div
        className="gap-patch"
      >
        <div
          className="circle"
        />
      </div>
      <div
        className="circle-clipper right"
      >
        <div
          className="circle"
        />
      </div>
    </div>
  </div>
</div>
`;
